tests/test_sparql_store.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from store.base import SparqlStore
+from SPARQLWrapper import QueryResult
+
+
+def test_get_datasets():
```

Comment:
you can make this `test_run sparql` and just test the method I've asked you to write this way.

you'll need a second test the to check that `get_datasets()` is returning a nice standard dict.